extensions/opentelemetry/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/OpenTelemetryConfig.java

```diff
@@ -15,6 +17,16 @@
     @ConfigItem(defaultValue = "true")
     public boolean enabled;
 
+    /**
+     * Comma separated list of OpenTelemetry propagators which must be supported.
+     * <p>
+     * Valid values are {@code b3, b3multi, baggage, jaeger, ottrace, tracecontext, xray}.
+     * <p>
+     * Default value is {@code traceContext}
+     */
+    @ConfigItem(defaultValue = "tracecontext")
```

Comment:
Do we need the default to be `tracecontext,baggage`?

I think I saw that's the default for OTeL itself

Reply:
Yup, just set it to `tracecontext,baggage`